games-emulation/hatari/hatari-2.1.0.ebuild

```diff
@@ -55,13 +56,15 @@ src_configure() {
 	mycmakeargs=(
 		"-DDOCDIR=/usr/share/doc/${PF}"
 		"-DENABLE_SDL2=$(usex sdl2)"
+		"-DENABLE_CAPSIMAGE5=ON"
 		)
 	cmake-utils_src_configure
 }
 
 src_install() {
 	cmake-utils_src_install
 	python_fix_shebang "${ED%/}"/usr/share/hatari/{hatariui,hconsole}/
+	readme.gentoo_create_doc
```

Comment:
Hmm, you're apparently fixing readme.doc use as well. This normally belongs in a separate commit but I can let you go with it as-is this time.